### PR TITLE
In game Info panel

### PR DESCRIPTION
**Pull request by warpdragon**
started a basic info panel.

### CODE REVIEW

**Comment by warpdragon** [comment]
image

**Comment by warpdragon** [comment]
Behold the Infopanel in all its glory!!!

**Comment by Haganeko** [comment]
Your farming cave description has a "which" clause not separated by commas. Otherwise, nice!

**Comment by Chronophylos** [comment]
cool

**Comment by zelding** [comment]
maybe creep-killing abilities? instead of properties?

**Comment by imaGecko** [comment]
the - in upper-hand doesn't need to be there

**Comment by chrisinajar** [comment]
Both farming and reflex say first, and the farming one says first twice

File: game/panorama/localization/addon_english.txt [hunk 1]
```diff
@@ -17,4 +17,20 @@
...
+
+  "game_info_header_objective"      "Objective:"
+  "brief_description"               "Accumulate points by killing players and bosses. First to hit the kill limit wins!"
+  "game_info_header_how_to_play"    "How To Play:"
+
+  "game_info_duels"                "Take to the <span class='duels'>Duel</span> Arena in a variety of team comps and battle to the death. Activate the highground and use the shop to get the upper-hand."
+
+  "game_info_bosses"               "<span class='bosses'>Bosses</span> and their loot are the key to unlocking powerful item upgrades. Their difficulty increases as you defeat them, and some don't play by the rules."
+
+  "game_info_farming_core"           "Killing the first Boss near your base for the first time rewards your team with a <span class='farmingcore'>Farming Core.</span> When combined with your upgraded boots, this item grants your hero special creep-killing properties."
```
Comment (chrisinajar) [inline game/panorama/localization/addon_english.txt]: "The first time you kill one of the bosses closest to your base, ..."

File: game/panorama/localization/addon_english.txt [hunk 2]
```diff
@@ -17,4 +17,20 @@
...
+
+  "game_info_reflex_core"          "Killing the second Boss near your base for the first time rewards your team with a <span class='reflexcore'>Reflex Core.</span> The Reflex items are among the most powerful items in the game."
```
Comment (chrisinajar) [inline game/panorama/localization/addon_english.txt]: I see what you mean here but it's worded really weird. Perhaps here, immediately after mentioning the other boss, could be a good spot to also introduce the concept of tiers. something like "When you kill the other tier 1 boss nearest to your base,..."

Comment (chrisinajar) [inline game/panorama/localization/addon_english.txt]: `The Reflex items are among the most powerful items in the game.` -> `The Reflex items are among the most powerful in the game.`

**Comment by warpdragon** [comment]
Leme take a look at it. 1 sec.

**Comment by warpdragon** [comment]
That should do it

File: game/panorama/localization/addon_english.txt (hunk 1 above)
Comment (warpdragon) [inline game/panorama/localization/addon_english.txt]: taken care of.

File: game/panorama/localization/addon_english.txt (hunk 2 above)
Comment (warpdragon) [inline game/panorama/localization/addon_english.txt]: taken care of.

**Pull request merged**